### PR TITLE
[UI] [Android] Make style of download card match the rest of the UI

### PR DESCRIPTION
**Pull request by endim8**
Makes the style of the download card match the rest of the UI:
new:
signal-2022-10-26-135207_002

### CODE REVIEW

**Comment by endim8** [comment]
Fixed dark mode and landscape:
Screenshot_20221026_142408

**Review by biodranik — CHANGES_REQUESTED** [review]

**Comment by endim8** [comment]
@biodranik PTAL

File: android/res/values/themes-base.xml [hunk 1]
```diff
@@ -154,7 +154,6 @@
 
     <item name="alertDialogTheme">@style/MwmTheme.Night.AlertDialog</item>
     <item name="windowBackgroundForced">@color/bg_window_night</item>
-    <item name="cardFrame">@drawable/card_frame_night</item>
```
Comment (biodranik) [inline android/res/values/themes-base.xml:157]: These pngs should be deleted if they're not used.

File: android/res/layout/onmap_downloader.xml [hunk 2]
```diff
@@ -16,8 +16,10 @@
     android:clipChildren="false"
     android:layout_gravity="center"
     android:gravity="center_horizontal"
-    android:background="?cardFrame"
+    android:backgroundTint="?menuBackground"
+    android:background="@drawable/card_background"
```
Comment (biodranik) [inline android/res/layout/onmap_downloader.xml:20]: Why card_background if it's onmap_downloader_background?

**Review by biodranik — CHANGES_REQUESTED** [review]

File: android/res/layout/onmap_downloader.xml (hunk 2 above)
Comment (endim8) [reply android/res/layout/onmap_downloader.xml]: renamed 👍

File: android/res/values/themes-base.xml (hunk 1 above)
Comment (endim8) [reply android/res/values/themes-base.xml:157]: I wasn't sure if they were still used, but I didn't realise android studio has safe delete! Removed them now 👍

**Comment by endim8** [comment]
@biodranik PTAL again :)

**Comment by endim8** [comment]
removed the unused stuff at the bottom of the view, looks much better now:
new

**Comment by endim8** [comment]
@biodranik final PTAL hopefully ;)

File: android/res/layout-land/onmap_downloader.xml [hunk 3]
```diff
@@ -21,84 +22,59 @@
...
+      <TextView
+        android:id="@+id/downloader_size"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/margin_half"
+        android:textAppearance="@style/MwmTextAppearance.Body2"
+        android:gravity="center_horizontal"
+        tools:text="1000 MB"/>
       <FrameLayout
-        android:id="@+id/banner_frame"
```
Comment (biodranik) [inline android/res/layout-land/onmap_downloader.xml:99]: An interesting place to show some hints for users!

**Review by biodranik — APPROVED** [review]

**Comment by vng** [comment]
Ward wrap is different in landscape/portrait modes. Card should be _rectangle_ in portrait, like in landscape:

telegram-cloud-photo-size-2-5408982002293653312-y

telegram-cloud-photo-size-2-5408982002293653313-y

**Comment by endim8** [comment]
> Ward wrap is different in landscape/portrait modes. Card should be _square_ in portrait, like in landscape:
> 
> telegram-cloud-photo-size-2-5408982002293653312-y
> 
> telegram-cloud-photo-size-2-5408982002293653313-y

fixed in https://github.com/organicmaps/organicmaps/pull/3793 :)

**Pull request merged**